README.md

```diff
@@ -1 +1,190 @@
-# rails-purse-app
\ No newline at end of file
+<a name="readme-top"></a>
+<div align="center">
+
+  <h1><b>Rails Purse App</b></h1>
+
+</div>
+
+# 📗 Table of Contents
+
+- [📗 Table of Contents](#-table-of-contents)
+- [📖 Rails Purse ](#-rails-purse-)
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+    - [Key Features ](#key-features-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Usage](#usage)
+    - [Tests](#tests)
+  - [👥 Author ](#-author-)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [📝 License ](#-license-)
+
+# 📖 Rails Purse <a name="about-project"></a>
+
+This is a **Rails Purse App** created with Ruby on Rails + Postgresql, as a Rails Solo Capstone Project.
+
+This is a transaction app, where users can sign up to organize expenses into different group, and add completed purchases for each created group, where they can see how much money they sent on each group. 
+
+Without logging in or signing up no pages are publicly available. 
+
+[Demo](https://rails-purse.onrender.com)
+
+[Video](https://www.loom.com/share/01651a1c5aed4efe98acc22c4670abd4?sid=6a9d231f-d403-4866-8159-9618f0c38847)
+
```

Comment:
- [ ] While you did well recording and even having it within 5 minutes, I think the audio of the recorder has issues, I could not hear anything from the beginning to the end, I will encourage you to record another one that can be heard.